docs/source/release/v3.13.0/muon.rst

```diff
@@ -8,18 +8,12 @@ MuSR Changes
 Interface
 ---------
 
-
-Interface
----------
-
-
 Improvements
 ############
 
 Bug fixes
 #########
 
-- Results table can now detect sequential fits.
```

Comment:
Was this release note meant to be removed?

Reply:
It was not meant to be removed.